static/app/components/searchQueryBuilder/tokens/filter/valueCombobox.tsx

```diff
@@ -842,9 +844,16 @@ export function SearchQueryBuilderValueCombobox({
     useMemo(() => {
       if (!showDatePicker) {
         return function (props) {
+          // Removing the ask seer options from the value list box props as we don't
+          // display and ask seer option in this list box.
+          const hiddenOptions = new Set(props.hiddenOptions);
+          hiddenOptions.delete(ASK_SEER_ITEM_KEY);
```

Comment:
Okay sorry one more followup here. Can we prevent the seer options from being added to `items` in the first place? `hiddenOptions` is created from the list of items, and if we never want to display them in this dropdown why are they included?

Reply:
No worries!

So they're being added to the list so that the combobox will register them as a valid option, and when using arrow keys to select them for instance it actually works properly. The reason why they need to be hidden by being added to `hiddenOptions` is because we don't want the default list item to render it, and instead we want to render out own component for "Ask Seer".